lib/error-handler.js

```diff
@@ -97,7 +97,9 @@ function fallbackErrorHandler (error, reply, cb) {
   const statusCode = reply.statusCode
   let payload
   try {
-    const serializerFn = getSchemaSerializer(reply[kRouteContext], statusCode, reply[kReplyHeaders]['content-type'])
+    let serializerFn = getSchemaSerializer(reply[kRouteContext], statusCode, reply[kReplyHeaders]['content-type'])
+    serializerFn = typeof serializerFn === 'object' ? Object.values(serializerFn)[0] : serializerFn
```

Comment:
Calling `Object.values(serializerFn)[0]` would not guarantee to pick a `JSON` one. The order is not guaranteed.